packages/tasit-sdk/src/Decentraland.forkedRopsten.test.js

```diff
@@ -0,0 +1,250 @@
+import { expect, assert } from "chai";
+
+import { Account, Action } from "./TasitSdk";
+const { ERC20, ERC721, Marketplace, ConfigLoader } = Action;
+const { Mana } = ERC20;
+const { Estate, Land } = ERC721;
+const { Decentraland: DecentralandMarketplace } = Marketplace;
+import config from "./config/default";
+
+import { ropsten as ropstenAddresses } from "../../tasit-contracts/decentraland/addresses";
+const {
+  MarketplaceProxy: MARKETPLACE_ADDRESS,
+  LANDProxy: LAND_ADDRESS,
+  MANAToken: MANA_ADDRESS,
+  EstateProxy: ESTATE_ADDRESS,
+} = ropstenAddresses;
+
+import {
+  createSnapshot,
+  revertFromSnapshot,
+  confirmBalances,
+  gasParams,
+  setupWallets,
+  addressesAreEqual,
+  bigNumberify,
+  etherFaucet,
+  ropstenManaFaucet,
+  constants,
+  ProviderFactory,
+  DecentralandUtils,
+} from "./testHelpers/helpers";
+
+const { ONE, TEN } = constants;
+
+const ROPSTEN_NETWORK_ID = 3;
+
+describe("Decentraland tasit app test cases (ropsten)", () => {
+  let ownerWallet;
+  let ephemeralWallet;
+  let manaContract;
+  let landContract;
+  let estateContract;
+  let marketplaceContract;
+  let landForSale;
+  let estateForSale;
+  let snapshotId;
+  let provider;
+
+  before("", async () => {
+    ConfigLoader.setConfig(config);
+
+    provider = ProviderFactory.getProvider();
+
+    const network = await provider.getNetwork();
+    const { chainId } = network;
+    expect(chainId, "The target network isn't ropsten.").to.equal(
+      ROPSTEN_NETWORK_ID
+    );
+
+    manaContract = new Mana(MANA_ADDRESS);
+    landContract = new Land(LAND_ADDRESS);
+    estateContract = new Estate(ESTATE_ADDRESS);
+    marketplaceContract = new DecentralandMarketplace(MARKETPLACE_ADDRESS);
+
+    const decentralandUtils = new DecentralandUtils();
+    const { getOpenSellOrders } = decentralandUtils;
+
+    const fromBlock = 0;
+    const openSellOrders = await getOpenSellOrders(fromBlock);
+
+    // Note: The exact amount of land isn't predictable since we are forking from the latest block
+    expect(openSellOrders).to.not.be.empty;
+
+    // Pick two open sell orders: one for a parcel of land and one for an estate
+    for (let sellOrder of openSellOrders) {
+      const { values: order } = sellOrder;
+      const { nftAddress, expiresAt } = order;
+
+      const isLand = addressesAreEqual(nftAddress, LAND_ADDRESS);
+      const isEstate = addressesAreEqual(nftAddress, ESTATE_ADDRESS);
+      const expired = Number(expiresAt) < Date.now();
+
+      // All parcels of land and estates for sale are expired (block 5058416) - otherwise we would select one that isn't expired
```

Comment:
```suggestion
      // All parcels of land and estates for sale are expired (block 5058416) -
      // otherwise we would select one that isn't expired
```